wiki/Article_styling_criteria/Writing/de.md

```diff
@@ -287,6 +282,18 @@ Artikel sollten einfache deutsche Grammatik verwenden.
 
 [Artikel mit informellen Registern](#sprachregister) sollten Semikolons (`;`) nicht verwenden.
 
+### Geviertstrich
```

Comment:
This is where the two ASC articles differ. Em dashes aren't really common in German. Instead, just an en dash or a minus is used. There are a few articles in the German wiki that use em dashes but I think we should stick to what is common (maybe change these articles to use common dashes too). The French ASC for example also uses `osu! (the game)` instead of `osu!—the game—`.

FYI: Wikipedia calls em dahes [`Geviertstrich`](https://de.wikipedia.org/wiki/Geviertstrich).

Reply:
Other languages should follow their own writing (non-markdown) rules, such as punctuation, dates, etc. Note how the [French article](https://osu.ppy.sh/wiki/fr/Article_styling_criteria/Writing) has a disclaimer at the top that states that the article only concerns English.

The ASC articles are originally intended to be a reference for English articles. If ASC/Writing was to be fully localised to different languages, they'd have to be in separate articles and require a lot of discussion between native contributors, so I doubt it'd happen at least for now. For that reason I recommend you do the same thing as the French translations and just ignore the parts of the ASC that don't apply to German when writing or reviewing.

> The French ASC for example also uses osu! (the game) instead of osu!—the game—.

You can absolutely just do the same in this case, or whatever makes sense.

Reply:
> Note how the [French article](https://osu.ppy.sh/wiki/fr/Article_styling_criteria/Writing) has a disclaimer at the top that states that the article only concerns English.
Does the strict content parity rule also need to be applied in a sense that it should describe what formats the English wiki uses (for example British English), or is it okay to leave them out and/or add language specific rules like the one above in line 285?

Reply:
Yes, parity applies here. I haven't checked the other translations, but I don't think they add or change anything language-specific. Either way, that would be out of scope for the asc.

If there's enough material to warrant language-specific asc's, then maybe idk, perhaps make an issue. That would be a better solution than burying some random info in translations of the asc.